test/src/test/java/org/corfudb/infrastructure/ManagementServerTest.java

```diff
@@ -26,7 +26,7 @@
     public ManagementServer getDefaultServer() {
         // Adding layout server for management server runtime to connect to.
         ServerContext serverContext = new ServerContextBuilder()
-                .setSingle(true)
+                .setSingle(false)
```

Comment:
Why change this?
If it is single mode then triggerFailureHandler actually fails with a wrong cluster exception.